uplink/piecestore/verification.go

```diff
@@ -14,13 +15,17 @@ import (
 	"storj.io/storj/pkg/signing"
 )
 
+const pieceHashExpiration = 2 * time.Hour
```

Comment:
should this be configurable or okay as const?

Reply:
It was and it still is a const in the satellite part, so I applied the same.

https://github.com/storj/storj/blob/master/satellite/metainfo/metainfo.go#L36

Which is used in https://github.com/storj/storj/blob/master/satellite/metainfo/validation.go#L378